spec/services/description_generator_spec.rb

```diff
@@ -34,6 +34,10 @@
         { name: [{ value: contrib1_name }], role: [{ value: 'Contributing author' }], type: 'person' },
         { name: [{ value: contrib2_name }], role: [{ value: 'Contributing author' }], type: 'person' },
         { name: [{ value: contrib3_name }], role: [{ value: 'Contributing author' }], type: 'person' }
+      ],
+      relatedResource: [
+        { identifier: [{ uri: 'http://my.awesome.research.io', displayLabel: 'My Awesome Research' }] },
```

Comment:
It might be nice to have an example that shows a RelatedLink with a missing title. Worth making factory & spec tweaks for this?

Reply:
good call, will do...